android/app/src/main/java/app/organicmaps/downloader/DownloaderFragment.java

```diff
@@ -54,9 +55,6 @@ public void onScrollStateChanged(RecyclerView recyclerView, int newState)
 
   private final MapSearchListener mSearchListener = new MapSearchListener()
   {
-    // Called from JNI.
-    @Keep
-    @SuppressWarnings("unused")
```

Comment:
Is it intended?

Reply:
Yes, It's an interface implementation. I can understand why is it written in the interface definition but not here

Reply:
So warnings are not an issue anymore?